src/vs/workbench/contrib/terminal/browser/terminalMenus.ts

```diff
@@ -709,7 +709,7 @@ export function getTerminalActionBarArgs(location: ITerminalLocationOptions, pro
 } {
 	let dropdownActions: IAction[] = [];
 	let submenuActions: IAction[] = [];
```

Comment:
makes sure we don't show cygwin/windowsPowerShell in the dropdown so as not to clutter it/overwhelm